services/shotgrid_common/utils.py

```diff
@@ -219,6 +219,7 @@ def create_ay_fields_in_sg_project(
             attribute containing the type of data and the scope of the
             attribute.
     """
+    # NOTE: this is not needed?
```

Comment:
```suggestion
```

Reply:
wait, I added that `NOTE` myself to discuss whether the code following that was required. I believe that is not since `SG_PROJECT_ATTRS` includes ALL the types of the queries you are doing so you don't need to construct that `ayon_attribs_mapping`?

Reply:
Pinging @jakubjezek001